content/en/sensitive_data_scanner/_index.md

```diff
@@ -119,28 +172,91 @@ You can create custom scanning rules using regex patterns to scan for sensitive
+    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of FarmHash.
+1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: If no tags are added when saving the rule, the Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.
+1. Click **Save**.
 
 **Notes**:
 
-- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
-- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
-- To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.
+- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
+- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/agent
+[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
+{{% /tab %}}
+{{< /tabs >}}
 
 See [Investigate Sensitive Data Issues][8] for details on how to use the [Summary][9] page to triage your sensitive data issues.
 
+### Edit scanning rules
+
+{{< tabs >}}
+{{% tab "In the Cloud" %}}
+
+1. Navigate to the [Sensitive Data Scanner][1] configuration page.
+1. Hover over the scanning rule you want to edit and click the **Edit** (pencil) icon.
+
+   The **Define match conditions** section shows either the regular expression you wrote for your custom rule or an explanation of the library scanning rule you chose along with examples of matched sensitive information.
+1. To make sure that a rule matches your data, you can provide a sample in the **Add sample data** section. If the rule finds matches in the sample data, a green **Match** label appears next to the input field.
+1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like "visa", "credit", and "card". You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.
```

Comment:
Use code formatting (eg. `visa`) for keywords, to be consistent with existing format (https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/sds-scanning-rule.md?plain=1#L1)

```suggestion
1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.
```